doc/nrf/ug_nrf9160_gs.rst

```diff
@@ -98,7 +98,7 @@ If you experience any problems during the process, restart the Programmer app by
 
    If the three COM ports are not visible, press ``Ctrl+R`` in Windows or ``command+R`` in macOS to restart the Programmer application.
 
-   The button text changes to the type of the selected device, with the SEGGER ID below the name.
+   The button text changes to the type of the selected device, with the device ID below the name.
```

Comment:
```suggestion
   The button text changes to the type of the selected device, with the SEGGER ID below the name.
```

I believe this should be `SEGGER ID`, referring to ID of the external SEGGER debugger on the nRF9160DK.